pipeline.yml

```diff
@@ -47,6 +48,13 @@ jobs:
             common/decrypted-development-bosh-main.yml \
             common/decrypted-development-bosh-external.yml \
             > bosh-manifest/manifest.yml
+  - task: lint-manifest
```

Comment:
WDYT about using a yaml anchor here to avoid copy and paste?

```yaml
- &lint-manifest
  task: lint-manifest
  file: ...
```

and later...

```yaml
- *lint-manifest
```